src/agents/pnp/daemon/osconfig.json

```diff
@@ -59,6 +59,10 @@
       "ComponentName": "Networking",
       "ObjectName": "NetworkConfiguration"
     },
+    {
+      "ComponentName": "PackageManagerConfiguration",
+      "ObjectName": "State"
+    },
     {
```

Comment:
You need to pull from main. This changed since this PR started. All object names are now camelCased instead PascalCased. Same for settings and settings values.